components/Notifications/Notifications.tsx

```diff
@@ -0,0 +1,39 @@
+import { Portal } from "@reach/portal";
+import { animated, useTransition } from "@react-spring/web";
+import { useNotificationsContext } from "hooks";
+import styled from "styled-components";
+import { Notification } from "./Notification";
+
+export function Notifications() {
+  const { notifications, removeNotification } = useNotificationsContext();
+
+  const transitions = useTransition(notifications, {
```

Comment:
I've used the same super basic enter/leave animation for the notifications and tooltips for now. its definitely good enough but it can be improved later.

Reply:
It's the same library that's used to animate the `Panel` component. It let's us be aware of when a component is shown or hidden, and add enter/leave transitions to that. This particular library [`react-spring`](https://react-spring.dev/) is one of the most popular animation libraries going around. It's advantage is that it uses spring-based physics to produce more natural looking animations than what can be achieved with css transitions alone.